docs/intro_06_audits.md

```diff
@@ -10,6 +10,7 @@ We take great care ensuring security of our smart contracts. Please find the aud
 - [Gnosis Safe v1.0.0](https://github.com/gnosis/safe-contracts/blob/v1.1.1/docs/Gnosis_Safe_Formal_Verification_Report_1_0_0.pdf).
 - [Gnosis Safe v1.1.0 & v1.1.1](https://github.com/gnosis/safe-contracts/blob/v1.1.1/docs/audit_1_1_1.md).
 - [Gnosis Safe v1.2.0](https://github.com/gnosis/safe-contracts/blob/v1.2.0/docs/audit_1_2_0.md).
+- [Gnosis Safe v1.3.0](https://github.com/gnosis/safe-contracts/blob/v1.3.0/docs/audit_1_3_0.md).
```

Comment:
This page doesn't exist for me. Getting a 404.

Reply:
Yes it is not created yet ;)